src/adapter/graphql/tests/tests/test_gql_account_flow_configs.rs

```diff
@@ -694,7 +696,9 @@ impl FlowConfigHarness {
             .bind::<dyn PollingIngestService, MockPollingIngestService>()
             .add::<DatasetOwnershipServiceInMemory>()
             .add::<DatasetOwnershipServiceInMemoryStateInitializer>()
-            .add::<DatabaseTransactionRunner>();
+            .add::<DatabaseTransactionRunner>()
+            .add::<InMemoryRebacRepository>()
+            .add::<RebacServiceImpl>();
```

Comment:
Do we still need these dependencies in unrelated tests?

Reply:
Nope -- thank you for noticing!

Removed